src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5252,14 +5255,27 @@ _pbs_python_event_set(unsigned int hook_event, char *req_user, char *req_host,
 		log_event(PBSEVENT_DEBUG3, PBS_EVENTCLASS_HOOK, LOG_INFO, __func__, log_buffer);
 	}
 
+	lval = enable_restart;
+	if (is_sattr_set(SVR_ATR_PythonRestartEnable))
+		enable_restart = get_sattr_long(SVR_ATR_PythonRestartEnable);
+	else
+		enable_restart = PBS_PYTHON_RESTART_ENABLE;
+	if (lval != enable_restart) {
+		snprintf(log_buffer, sizeof(log_buffer),
+			"python_disable_restart is now %ld", enable_restart);
```

Comment:
The log message talks about "python_disable_restart" but the attribute is "python_enable_restart".

Reply:
Apparently I missed one when I renamed the attribute to a more positive name.  Fixed in 97498d40fc6d249c395d7714dfc15ca99b53455a.

BTW, if the attribute is unset, line 5262 sets the default to enabling the interpreter restart to maintain backwards compatibility.  Do I need to set the default somewhere else as well?

Reply:
@toonen : ah I see, it's okay then.